### PR TITLE
Add 20i

### PR DESCRIPTION
**Pull request by cyberfountain**
Add 20i to Hosting and cdn category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -736,6 +737,7 @@ Table of Contents
 ## Web Hosting
 
   * [000WebHost](https://www.000webhost.com/) — Zero cost website hosting with Apache, PHP, MySQL, cPanel with ads on the bottom of each pages!
+  * [20i](https://www.20i.com/) — Free web hosting with no ads, on the same platform as paid plans. Includes a free CDN, 100 email addresses, SSL and over 80 one-click installs.
```
Comment (ripienaar) [inline README.md:740]: thanks, please add it to one catagory only

Comment (cyberfountain) [reply README.md:739]: Thanks, removed from cdn category :)

**Comment by ripienaar** [comment]
thanks

**Comment by thispsj** [comment]
> Add 20i to Hosting and cdn category.

@cyberfountain did you try this ? It may be a problem on my side only but I'm just stuck at the transfer/register domain page.

**Pull request merged**